app/grandchallenge/reader_studies/models.py

```diff
@@ -1441,6 +1447,33 @@ def allow_null_types(self):
             self.AnswerType.NUMBER,
         ]
 
+    @property
+    def allow_blank_types(self):
+        return [
+            self.AnswerType.TEXT,
+            self.AnswerType.SINGLE_LINE_TEXT,
+            self.AnswerType.MULTI_LINE_TEXT,
+        ]
+
+    @property
+    def allow_empty_list_types(self):
+        return [
+            self.AnswerType.MULTIPLE_CHOICE,
+            self.AnswerType.MULTIPLE_CHOICE_DROPDOWN,
+        ]
+
+    @property
+    def empty_answer_values(self):
+        """Returns a list of answer values which are considered to be empty"""
+        result = []
+        if self.answer_type in self.allow_null_types:
+            result.append(None)
+        if self.answer_type in self.allow_blank_types:
+            result.append("")
+        if self.answer_type in self.allow_empty_list_types:
+            result.append([])
+        return result
```

Comment:
There's a risk here that you still have multiple empty answer representations. I think it is better to build a map of answer type to empty answer representation, e.g.

```python
EMPTY_ANSWER_VALUES = {
  AnswerType.TEXT: "",
  AnswerType.MULTIPLE_CHOICE: [],
  ...
}

...

  def empty_answer_values(self):
    return EMPTY_ANSWER_VALUES[self.answer_type]
```

That way we ensure there is only one representation and it is clear to the developer what to expect. If a key is missing you can either leave it as is or make it explicit that the programmer needs to extend the map.